app/main.py

```diff
@@ -1 +1,52 @@
-# write your code here
+
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills.extend(["JavaScript", "HTML", "CSS"])
```

Comment:
self.skills = ["JavaScript", "HTML", "CSS"]

Reply:
I think, my solve is correct